contracts/database/gorm/wire_interface.go

```diff
@@ -18,10 +18,10 @@ func NewInitializeImpl() *InitializeImpl {
 	return &InitializeImpl{}
 }
 
-func (receive *InitializeImpl) InitializeGorm(config config.Config, connection string) *GormImpl {
-	return InitializeGorm(config, connection)
+func (receive *InitializeImpl) InitializeGorm(config config.Config, connection string) *gorm.GormImpl {
+	return gorm.InitializeGorm(config, connection)
 }
 
-func (receive *InitializeImpl) InitializeQuery(ctx context.Context, config config.Config, connection string) (*QueryImpl, error) {
-	return InitializeQuery(ctx, config, connection)
+func (receive *InitializeImpl) InitializeQuery(ctx context.Context, config config.Config, connection string) (*gorm.QueryImpl, error) {
+	return gorm.InitializeQuery(ctx, config, connection)
 }
```

Comment:
S: Move this back to `database/gorm/`

Reply:
> S: Move this back to `database/gorm/`

After the mock is moved back, it will not be scanned by Mockery.

Reply:
Only move `InitializeImpl ` back.